libwacom/libwacom.c

```diff
@@ -350,6 +351,8 @@ libwacom_copy(const WacomDevice *device)
 	d->num_buttons = device->num_buttons;
 	d->buttons = g_memdup (device->buttons, sizeof(WacomButtonFlags) * device->num_buttons);
 	d->button_codes = g_memdup (device->button_codes, sizeof(int) * device->num_buttons);
+	memcpy(&d->edid, &device->edid, device->num_edid * sizeof (*device->edid));
```

Comment:
Is `d->edid` being assigned/allocated somewhere? It looks like its value hasn't been set after the call to `g_new0`. Segfault waiting to happen?